src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -47,6 +47,8 @@ class NexusPublishPlugin : Plugin<Project> {
     companion object {
         // visibility for testing
         const val SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME = "closeAndReleaseStagingRepository"
+        const val SIMPLIFIED_CLOSE_TASK_NAME = "closeStagingRepository"
+        const val SIMPLIFIED_RELEASE_TASK_NAME = "releaseStagingRepository"
```

Comment:
```suggestion
        private const val SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME = "closeAndReleaseStagingRepository"
        private const val SIMPLIFIED_CLOSE_TASK_NAME = "closeStagingRepository"
        private const val SIMPLIFIED_RELEASE_TASK_NAME = "releaseStagingRepository"
```

There's no harm repeating the task names in test annotations! It actually gives more hardening quality to the tests, i.e. if someone changes the constants, tests will break, which is a good thing as they're public API.

Reply:
Sure, I just followed existing style, fixed now.